### PR TITLE
PWGMM: dndeta-hi.cxx modified 

### PR DESCRIPTION
**Pull request by hyungjun-lee**
* add centrality analysis code
* add pt distribution analysis code

### CODE REVIEW

File: PWGMM/Mult/Tasks/dndeta-hi.cxx [hunk 1]
```diff
@@ -71,23 +81,24 @@ static constexpr TrackSelectionFlags::flagtype trackSelectionTPC =
...
   Service<ccdb::BasicCCDBManager> ccdb;
   Configurable<std::string> path{"ccdb-path", "Users/s/sherrman/My/Object", "base path to the ccdb object"};
   Configurable<std::string> url{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
-  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
 
-  Configurable<int> MultBin{"MultBin", 100, "# of Multiplicity bin"};
-  Configurable<double> MultMin{"MultMin", 0, "minimum of Multiplicity bin"};
-  Configurable<double> MultMax{"MultMax", 10000, "maximum of Multiplicity bin"};
-  AxisSpec MultAxis = {MultBin, MultMin, MultMax};
+  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-hi.cxx:95]: This configurable and the three after it are not used anywhere in the code, why do you need them?

File: PWGMM/Mult/Tasks/dndeta-hi.cxx [hunk 2]
```diff
@@ -158,108 +157,97 @@ struct MultiplicityCounter {
...
-
-    registry.fill(HIST("Multiplicity"), multV0A, multT0A, multT0C);
 
+    registry.fill(HIST("Centrality"), cent);
     registry.fill(HIST("Events/Selection"), 1.);
+    auto z = collision.posZ();
+
     if (!useEvSel || collision.sel8()) {
-      registry.fill(HIST("Centrality_MBAND"), cent);
+      if (z > -10 && z < 10) {
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-hi.cxx:184]: You could write `std::abs(z) < 10`

**Review by aalkin — CHANGES_REQUESTED** [review]
Please remove any unused code or variables.

File: PWGMM/Mult/Tasks/dndeta-hi.cxx (hunk 1 above)
Comment (hyungjun-lee) [reply PWGMM/Mult/Tasks/dndeta-hi.cxx:95]: I don't know how to use the configuration of "ccdb". when I copy some code for making dndeta-hi.cxx, maybe this part is copied too. Can I skip the "ccdb" configuration?

Comment (aalkin) [reply PWGMM/Mult/Tasks/dndeta-hi.cxx:91]: You do not need to configure ccdb if you do not query it directly from your task. The configurables in your task are just special variables that can receive values externally, they do not configure anything unless you use them explicitly. You can safely remove these.

**Review by aalkin — APPROVED** [review]

**Pull request merged**